src/test/resources/ObsBundle.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bundle xmlns="http://hl7.org/fhir">
+ <type value="collection"/>
```

Comment:
Not sure if Hapi can accept a bundle with type `collection` . it should be a `transaction`